src/amber/cli/commands/run.cr

```diff
@@ -19,9 +19,13 @@ module Amber::CLI
       end
 
       def run
-        release = "--release" if options.e.downcase == "production"
+        build_options = Array(String).new
+        build_options << "--release" if options.e.downcase == "production"
+        build_options << "--no-debug" if options.e.downcase == "production"
+        build_options << " " if build_options.size > 0
```

Comment:
This may be a bit cleaner:

```crystal
if options.e.downcase == "production"
  build_options.concat ["--release", "--no-debug", " "]
end

```

Reply:
I wrote it this way so that we could add different options for different reasons but only have a space if there was at least one option.

If it will never change we could just write `build_options += "--release --no-debug "`

Reply:
Right, but `.downcase` and `==` are done twice, what is not really good

Reply:
I agree with the above is much cleaner. I will also remove the additional space `" "` and just add the space on line 28  between the `} $`

Reply:
But then it ends up with 2 spaces in the case that the env isn't production.